### DISCUSSION TITLE
Fallback to html when opening .svg, .vue, etc

### DISCUSSION THREAD

**Discussion by smastrom** [zed-industries/zed#6604] [Show and tell]
When opening .svg or .vue files, by default, Zed sets the language (bottom-right) to "Unknown".

I always have to manually select HTML from the language menu in order to get syntax highlighting and especially for .vue files even if not "native", it actually works very well.

I'm pretty sure other extensions which like .vue are just an HTML superset can benefit of a feature that either automatically fallbacks to HTML or allows specifying a supported language to be used for an extension.

**Comment by JosephTLyons** [comment]
Having the abillity to configure the settings to map certain file types to known languages would be helpful - Im going to push for this from the team:

- https://github.com/zed-industries/community/issues/595

**Comment by MrVintage710** [comment]
Yeah this would be nice. I definitely want to use Zed at my job, but there isn't a convenient way to use Zed currently for Vue.  But I could live just html error checking.
